### ISSUE TITLE
Put content in editor

### ISSUE THREAD

**Issue by topdown** [nhnent/tui.editor#190]
This is being used in a POST form system but I am using a normal div like the example as its Ajax anyway.
The issue is that I can not get my data back in the editor when going back. ( IE. from database)
I tried single and double quotes.
I also tried setMarkdown.

## Current Behavior

```js
// Write example code

var content = editor.setValue('Some**thing**
Testing
 
 &gt; **Foo Bar**
 &gt; This
 Foo bar');

console.log(content);
```

## Expected Behavior
I expect it to place the content back in the editor.
Instead I get.
```
Uncaught SyntaxError: Invalid or unexpected token
```
On line `var content = editor.setValue('Some**thing**`

**Comment by topdown** [comment]
I just realized that I need to parse the markdown back to HTML before putting it back in the editor.

**Comment by topdown** [comment]
Ok, I figured it out.
Parse it back to HTML, then strip the \n then add it to the editor.
```php
$parser  = new \markdown();
$content = str_replace( '&gt;', '>', $description );
$content = $parser->text( $content );
$content = str_replace("\n", '', $content);
```
Then the JS
```js
editor.setValue( '<?php echo $content; ?>' );
```

I think the biggest issue is that the documentation "Getting started" stops after the editor is loaded.
There is no documentation that I found after that to handle putting content back.
I realize this is probably do to the vast amount of languages that may be using this package, but a few hints in the docs would be helpful for others.
